spec/shared_examples/storage_adapter/shared_examples.rb

```diff
@@ -118,8 +121,10 @@
   before {
```

Comment:
I tried that a while back and ran into some issue that I can't remember now, but it's definitely worth revisiting.  It would be nice not to have to check like this.  I may ask you about that if I try it again.  Thanks!